packages/wonder-blocks-banner/src/components/__docs__/banner.stories.js

```diff
@@ -33,7 +33,20 @@ export default {
         ): any),
         docs: {
             description: {
-                component: null,
+                component: `Banner. A banner displays a prominent message and
+                    related optional actions. It can be used as a way of
+                    informing the user of important changes. Typically, it is
+                    displayed toward the top of the screen.\n\nThere are two
+                    possible layouts for banners - floating and full-width.
+                    The \`floating\` layout is intended to be used when
+                    there is whitespace around the banner. The \`full-width\`
+                    layout is intended to be used when the banner needs to be
+                    flush with surrounding elements.\n\n\n### Usage\n\`\`\`jsx
+
+                    import Banner from "@khanacademy/wonder-blocks-banner";
+                    \n<Banner\n\ttext="Here is some example text."\n\tkind="success"\n\tlayout="floating"\n\tactions={[\n\t\t{title: "Button 1", onClick: () => {}},\n\t\t{title: "Button 2", onClick: () => {}},\n\t]}\n\tonDismiss={() => {console.log("Has been dismissed.")}}\n/>
```

Comment:
I wonder if we should add a small third-party dep for this case (not sure if it's worth tho).

This could be a good option: https://www.npmjs.com/package/ts-dedent

wdyt?

Reply:
Ooo wait! This gives me an idea

Reply:
I made the description a const so that the indents don't have to be part of it. What do you think now?

Reply:
ohhhh that's even better 👏